pages/03.Tutorials/18.create-backup-and-restore/default.en.md

```diff
@@ -7,16 +7,17 @@ taxonomy:
         - kubernetes
         - cli
         - ark
```

Comment:
remove ark

Reply:
I left that there intentionally, so that users who search for ark will still find the backup page